types/src/database_sql.rs

```diff
@@ -615,6 +615,20 @@ impl Database {
 
     // pub async fn maybe_undefined_to_vec() -> Vec<Option<String>> {}
 
+    pub async fn add_bookmark(&self, document_id: Uuid, user_id: Uuid) -> Result<String> {
+        query_file!("queries/add_bookmark.sql", document_id, user_id)
+            .execute(&self.client)
+            .await?;
+        Ok(format!("document: {}, user: {}", document_id, user_id))
```

Comment:
We should return the `AnnotatedDoc` right?

Reply:
This matches with what we're doing in other places, such as update_contributor_attribution or update_word. We're returning ids.